### PR TITLE
Add missing code in mdx custom module example

### PR DESCRIPTION
**Pull request by remcohaszing**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/orgs/unifiedjs/discussions/205#discussioncomment-3895977
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by ChristianMurphy — APPROVED** [review]
Thanks @remcohaszing!

File: types/mdx/index.d.ts [hunk 1]
```diff
@@ -47,9 +47,14 @@
  * Create the following file named `my-component.mdx.d.ts` in the same directory:
```
Comment (wooorm) [inline types/mdx/index.d.ts:47]: Can you remove the word `additional` on L36?

File: types/mdx/index.d.ts [hunk 2]
```diff
@@ -47,9 +47,14 @@
...
  *
  * ```ts
+ * export { default } from '*.mdx';
+ *
  * export const message: string;
  * ```
  *
+ * Note that this means you also need to define the default export. You can also define your own
+ * default export type to narrow the accepted prop types.
```
Comment (wooorm) [inline types/mdx/index.d.ts:56]:
```suggestion
 * Note that this overwrites the `declare module '*.mdx' { … }` types from earlier, which 
 * is why you also need to define the default export. You can also define your own
 * default export type to narrow the accepted prop types of this specific file.
```

**Review by wooorm — COMMENTED** [review]
Thanks Remco! Your suggestion is already good, but I think these edits will make it more clear to people who run into this, what is happening!

**Review by ChristianMurphy — APPROVED** [review]

**Review by wooorm — APPROVED** [review]

**Review by ChristianMurphy — APPROVED** [review]

**Pull request merged**